### PR TITLE
Enforce alphabetically ordered cargo deps

### PR DESCRIPTION
**Pull request by dknopik**
Shamelessly stolen from @macladson's PR here: https://github.com/sigp/lighthouse/pull/6678

### CODE REVIEW

File: .github/workflows/test-suite.yml [hunk 1]
```diff
@@ -192,3 +192,18 @@ jobs:
 #          cache-target: release
 #      - name: Run Makefile to trigger the bash script
 #        run: make cli
+  cargo-sort:
```
Comment (diegomrsantos) [inline .github/workflows/test-suite.yml:195]: What do you think about moving this job and the `check-code` to a new workflow as they are more about code quality than testing?

Comment (dknopik) [reply .github/workflows/test-suite.yml:195]: Don't really have an opinion on that. @magick93, what do you think?

Comment (magick93) [reply .github/workflows/test-suite.yml:195]: IMO splitting it wont add too much value, though I'm happy to do it. 

The best value - in the case of test and code qual. comes from giving rapid feedback to the dev. It its not too computationally intensive maybe a pre-commit hook may be better.

**Review by jking-aus — APPROVED** [review]
lgtm

File: .github/workflows/test-suite.yml (hunk 1 above)
Comment (jking-aus) [reply .github/workflows/test-suite.yml:195]: good comment diego, let's merge for now but as workflows become more refined we can revisit

**Pull request merged**